nb2workflow/deploy.py

```diff
@@ -298,15 +308,40 @@ def _build_with_kaniko(git_origin,
         return container_metadata
 
 
-def _build_with_docker(git_origin, 
+def _extract_resource_requirements(local_repo_path, ontology_URL=default_ontology_URL):
+    sp.check_call([
+        "wget", ontology_URL, "-O", "ontology.ttl"
+    ])
+    ontology = Ontology("ontology.ttl")
+
+    local_repo_path = str(local_repo_path)
+    resources = {}
+
+    for nb_file in glob.glob(local_repo_path + '/**/*.ipynb', recursive=True):
```

Comment:
`os.path.join` would be safer depending on what is in `local_repo_path`

Reply:
Done